sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+friendly_name: Firefox Health Indicator Searches By Provider
+description: |-
+  Count of searches & users by provider/day used in Firefox Health Indicator dashboard
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: kwindau@mozilla.com
+  table_type: aggregate
```

Comment:
Should this also include the `shredder_mitigation` label?

Reply:
Good catch, added! thanks!